src/xdp/trn_transit_xdp.c

```diff
@@ -136,6 +137,35 @@ transit switch of that network, OW forward to the transit router. */
 	nkey.nip[2] = inner_dst_ip;
 	net = bpf_map_lookup_elem(&networks_map, &nkey);
 
+	/* Cache lookup for known ep */
+	struct remote_endpoint_t *dst_r_ep;
+	struct endpoint_key_t dst_epkey;
+	__builtin_memcpy(&dst_epkey.tunip[0], &tunnel_id, sizeof(tunnel_id));
+	dst_epkey.tunip[2] = inner_dst_ip;
+	dst_r_ep = bpf_map_lookup_elem(&ep_host_cache, &dst_epkey);
+
+	/* Only rewrite RTS and update cache when router host */
```

Comment:
minor remove "when router host", there is no actual router here.

Reply:
fixed